### DISCUSSION TITLE
Change font color depening on value/variable

### DISCUSSION THREAD

**Discussion by kieranwalsh** [EvotecIT/PSWriteHTML#359] [Q&A] [answered]
Is there a way that I can change font color depending on value? For example, this is part of a host output I'm converting to HTML where Os versions that are the latest version are shown in Green, older OSes that are still supported are Yellow, and out-of-date versions are shown in red: 
![image](https://user-images.githubusercontent.com/623629/208794136-2de4c4aa-72f0-4759-9e47-cc8b31ab04fe.png)

Here, I've got my data in HTML. I won't be using the "OS Font Colour" column. I've included it to show that I have the data stored as a varaible so I can use it in the "Operating System Version" column:
![image](https://user-images.githubusercontent.com/623629/208794534-e3ec943a-7617-45bf-9fbe-201ca01f12d4.png)

Here's part of the relevant code:
```
$ClientData = foreach($Client in $Clients)
{
    $OSName, $OSVersion, $OSFontColour = get-OSVersion -OSVersion $Client.OperatingSystemVersion
    [PSCustomObject]@{
        'Computer'                 = $Client.Name
        'Operating System Name'    = $Client.OperatingSystem
        'Operating System Version' = $OSName + $OSVersion
        'OS Font Colour'           = $OSFontColour
    }
}

```
and the HTMLContent:

```
New-HTMLContent -HeaderText 'Computer Details' -HeaderBackGroundColor 'DarkSlateGray' {
    New-HTMLPanel -Margin 10px {
         New-HTMLPanelStyle -BorderRadius 5px
         New-HTMLTable -Title 'Computer Details' -DataTable $ClientData -DataTableID 'ClientOperatingSystemsFull'
     }
}
```

Thanks for any suggestions.

**Comment by PrzemyslawKlys** [comment, marked as answer]
What you want is New-TableCondition (or New-HTMLTableCondition alias) that can also work within NewTableCondtionGroup. 

It's shown on this blog post:
- https://evotec.xyz/advanced-html-reporting-using-powershell/
- https://evotec.xyz/dashimo-easy-table-conditional-formatting-and-more/ (Dashimo is now part of PSWriteHTML)

You can also find it in the Examples folder on GitHub in large quantities.

**Reply by kieranwalsh** [reply]
That's fantastic - thanks!
